Apps/W1/ShopifyConnector/app/Inventory/Codeunits/ShpfyInventoryAPI.Codeunit.al

```diff
@@ -0,0 +1,267 @@
+/// <summary>
+/// Codeunit Shpfy Inventory API (ID 30195).
+/// </summary>
+codeunit 30195 "Shpfy Inventory API"
+{
+    Access = Internal;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"